src/stratigraphy/metadata/metadata.py

```
@@ -162,7 +162,7 @@ def from_json(cls, json_metadata: dict, filename: str) -> "BoreholeMetadata":
 
 
 @dataclass
-class BoreholeMetadataList(metaclass=abc.ABCMeta):
+class OverallBoreholeMetadata(metaclass=abc.ABCMeta):
```

Ruff

`OverallBoreholeMetadata` is an abstract base class, but it has no abstract methods

(B024)